views/signup.erb

```diff
@@ -0,0 +1,21 @@
+<h3>Sign up to Chitter</h3>
+<form action='/signup' method='post'>
+  <p>
```

Comment:
p tags can be inline which will make your code look a lot tidier on the page. Eg: <p> `<input type='text' name='firstname' placeholder='First Name'> </p>`

Reply:
Alternatively you could put all this in one div block and use the <br> tag between them which adds a line space